repaygithub/cactus
# CACTUS-397 :: DataGrid Customization

Ticket: https://repayonline.atlassian.net/browse/CACTUS-397

Currently, I don't have this set up to allow developers to control flex properties on `TopSection` or `BottomSection`, because I'd be worried that that could open the door for issues with device responsiveness. Do you guys think we should let them change things like `justify-content` for instance?

Note: I moved `<ScreenSizeProvider>` into the storybook cactus addon so that it's always rendered in storybooks. For this reason, I removed usage of `ScreenSizeProvider` in several other stories.


### Testing
1. Open the DataGrid basic usage story
2. Play with the different features like pagination, changing the page size, and sorting to make sure they still work.
3. Change the screen size and the `cardBreakpoint` and make sure everything still looks good and functions properly at different screens sizes.
4. Feel free to modify the story to move some components around and see if you think it should satisfy developer needs; using margin props should help with organizing component placement in the top/bottom flex components.
